depthai.py

```diff
@@ -131,6 +147,66 @@ def stream_type(option):
         stream_dict = {'name': stream_name, "max_fps": max_fps}
     return stream_dict
 
+def decode_tracklets(packetData):
+    packetData.dtype = np.int32
+    tracklets = []
+    nr_objects = packetData[0]
+    tracklet_base = packetData[2:]
+    tracklet_size = 8
```

Comment:
This could be included in the Tracklet class and is a great example of a class variable (shared between all instances).